.github/actions/cross-branch-migration-test/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Cross-branch migration test
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check out master request branch
+      uses: actions/checkout@v4
+      with:
+        ref: master
```

Comment:
how about you introduce an input variable for the `base` ref so we can use this on release branches as well. It would also allow us to test migrations across release branches. 🤩

Reply:
good idea!